File: nodejs/sendSms.js
// Your accountSid and authToken from twilio.com/user/account
var accountSid = 'ACb4601ebfba5a649c4aae1ccbf6b1ee8f';
var authToken = "a4d6a5026b373ccd8cd7405d8026adfa";
var client = require('twilio')(accountSid, authToken);
 
client.messages.create({
    body: "Jenny please?! I love you <3",
    to: "+16163890356",
    from: "+17038280845"
}, function(err, message) {
    process.stdout.write(message.sid);
});

File: twillio/send_sms.py
# Download the twilio-python library from http://twilio.com/docs/libraries
from twilio.rest import TwilioRestClient

# Find these values at https://twilio.com/user/account
account_sid = "ACb4601ebfba5a649c4aae1ccbf6b1ee8f"
auth_token = "a4d6a5026b373ccd8cd7405d8026adfa"
client = TwilioRestClient(account_sid, auth_token)

message = client.messages.create(to="+17033444899", from_="+17038280845",
                                     body="Hello there!")
